lib/image_helper.rb

```diff
@@ -1,5 +1,5 @@
 module ImageHelper
-  CDN_URL = '//assets.craftacademy.se'
+  CDN_URL = 'https://assets.craftacademy.se'
```

Comment:
Was it necessary to add the `https` - I usually leave it out and just put `//` this way the browser will use the protocol that the site was originally served on. But I guess it doesn't matter now since we're serving those assets over `https` anyway..